src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -50,29 +50,12 @@ internal class UnifiedSuggestedActionsSource
                 document, selection, includeSuppressionFixes, isBlocking,
                 addOperationScope, cancellationToken), cancellationToken).ConfigureAwait(false);
 
-            var filteredFixes = FilterOnAnyThread(fixes);
+            var filteredFixes = fixes.Where(c => c.Fixes.Length > 0).ToImmutableArray();
```

Comment:
```suggestion
            var filteredFixes = fixes.WhereAsArray(c => c.Fixes.Length > 0);
```

Reply:
Let's find out! I made these changes in VS Code, and have discovered that I've been ruined by nice IDEs and am completely incapable of reasoning about code without them.

I even created this PR as a draft first because I wasn't confident 😛